code/modules/power/tesla/energy_ball.dm

```diff
@@ -12,17 +21,28 @@
 	move_self = 1
 	grav_pull = 0
 	contained = 0
```

Comment:
Should these be `TRUE`/`FALSE`?

Reply:
I don't know what this even does; I'll remove the var---I"m guessing this is some really old var that never got purged.